packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.5.0+2
+
+* Add a dependency on kotlin-bom to align versions kotlin transitive dependencies.
```

Comment:
Nit: Adds

https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style